README.md

```diff
@@ -52,6 +52,25 @@ Next follow these instructions.
 5. To run the migrations run: `python manage.py migrate`
 6. You can terminate the process by `Ctrl+C` in your terminal.
 
+### Run with Docker
+1. Make sure latest version of docker-compose is installed by running ``docker-compose -v``. If it is not,install using ``pip install -U docker-compose``
+
+2. Change Database HOST in ``settings.py``
+
+- Go to ``main/settings.py``
+- Under databases change host to db.
+
+3. **Build docker image**
+
+Execute ``sudo docker-compose run web`` to build the image.
+
+4. **Run docker image**
+
+Run ``sudo docker-compose up`` from the root directory of project.
```

Comment:
as step 4 you could have an additional step with the link where the server is running, so we don't have to look at dockerfiles to test it live. is it localhost:8000?

Reply:
Yes it is localhost:8000